test/test-applications/integrations/TestApplication.MongoDB/TestApplication.MongoDB.csproj

```diff
@@ -5,23 +5,19 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <ApiVersion Condition="'$(ApiVersion)' == ''">2.13.3</ApiVersion>
-    <DefineConstants Condition="'$(ApiVersion)'>='2.15.0'">$(DefineConstants);MONGODB_2_15</DefineConstants>
+    <TestedPackageVersion Condition="'$(TestedPackageVersion)' == ''">2.13.3</TestedPackageVersion>
```

Comment:
just `PackageVersion` instead of `TestedPackageVersion` ?

Reply:
`PackageVersion` is used by https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/3d3f5c395ac6eb0a2d774f25c87c9f74b2a8f266/test/Directory.Packages.props#L4
Even if it is tenically possible, I would like to avoid colision name.

I am open for the better naming, but I do not like `ApiVersion`.

Reply:
What about `LibraryVersion`?

Reply:
Fixed in b85b07a9c43c743d1c442b679d963f7f57da6b1b
Technically LibVersion can be different than PackageVersion but it is probably better naming.